qt/python/mantidqt/mantidqt/widgets/helpwindow/helpwindowbridge.py

```diff
@@ -0,0 +1,68 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2017 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+import sys
+from mantidqt.widgets.helpwindow.helpwindowpresenter import HelpWindowPresenter
+from qtpy.QtCore import QUrl
+from qtpy.QtWidgets import QApplication
+
+
+_presenter = None
+
+
+def show_help_page(relative_url, local_docs=None, online_base_url="https://docs.mantidproject.org/"):
+    """
+    Show the help window at the given relative URL path
+    """
+    global _presenter
+    if _presenter is None:
+        _presenter = HelpWindowPresenter()
+
+    if not relative_url or not relative_url.endswith(".html"):
+        relative_url = "index.html"
+
+    if local_docs and os.path.isdir(local_docs):
+        # Use local docs
+        full_path = os.path.join(local_docs, relative_url)
+        file_url = QUrl.fromLocalFile(full_path)
+        _presenter.view.browser.setUrl(file_url)
+    else:
+        # Use online docs
+        full_url = online_base_url.rstrip("/") + "/" + relative_url
+        _presenter.view.browser.setUrl(QUrl(full_url))
+
+    _presenter.show_help_window()
+
+
+def main(cmdargs=None):
```

Comment:
```suggestion
def main(cmdargs=sys.argv):
```
then you can simplify L55 below